packages/kbn-storybook/src/webpack.config.ts

```diff
@@ -202,4 +201,9 @@ export default function ({ config: storybookConfig }: { config: Configuration })
     },
     config
   );
+  // webpackMerge is concatenating the storybookConfig and config
```

Comment:
I'm somewhat surprised this was working as is, the merged config doesn't look how I would expect it to.  This needs a better fix, but assuming it works I'll leave it for the reviewers judgement on priority.